From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Samuel Attard <marshallofsound@electronjs.org>
Date: Sun, 5 May 2024 16:07:58 -0700
Subject: fix: use app_launch_prefetch namespace for SubprocessType

This should be upstreamed, looks like Chromium doesn't have a target that builds this file?

diff --git a/content/browser/ppapi_plugin_process_host.cc b/content/browser/ppapi_plugin_process_host.cc
index e8203ff5470f91ac35a5f444f6598866e75f88bb..b046a430eee010ca634f019ece3b0394299703db 100644
--- a/content/browser/ppapi_plugin_process_host.cc
+++ b/content/browser/ppapi_plugin_process_host.cc
@@ -237,7 +237,7 @@ bool PpapiPluginProcessHost::Init(const ContentPluginInfo& info) {
 
 #if BUILDFLAG(IS_WIN)
   cmd_line->AppendArgNative(
-      app_launch_prefetch::GetPrefetchSwitch(SubprocessType::kPpapi));
+      app_launch_prefetch::GetPrefetchSwitch(app_launch_prefetch::SubprocessType::kPpapi));
 #endif  // BUILDFLAG(IS_WIN)
 
   // These switches are forwarded to plugin pocesses.
